Lambdas/GHOAuth/OAuthLambda.swift

```diff
@@ -53,20 +57,27 @@ struct GHOAuthHandler: LambdaHandler {
         let userID: Int
 
         do {
-            userID = try await getUserID(accessToken: accessToken)
+            userID = try await getGHUserID(accessToken: accessToken)
         } catch {
             logger.error("Error getting user ID: \(error)")
             return .init(statusCode: .badRequest, body: "Error getting user")
         }
 
+        let jwt: GHOAuthPayload
+
         do {
-            let jwt = try await verifyState(state: String(event.queryStringParameters?["state"] ?? ""))
+            jwt = try await verifyState(state: String(event.queryStringParameters?["state"] ?? ""))
         } catch {
             logger.error("Error during state verification: error: \(error), state: \(event.queryStringParameters?["state"] ?? "")")
             return .init(statusCode: .badRequest, body: "Error verifying state")
         }
 
-        // TODO: Link id to Discord user
+        do {
+            try await userService.linkUser(discordID: jwt.discordID.rawValue, githubID: userID.description)
```

Comment:
Swift team advices to use direct string interpolation instead of `.description`. I'm not sure why, maybe it's easier for the compiler to optimize that code:
```suggestion
try await userService.linkUser(discordID: jwt.discordID.rawValue, githubID: "\(userID)")
```